Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -339,7 +339,7 @@ The property contains the names of the manipulated tables as keys and a map of o
 
    $cmd['tt_content'][1203]['copy'] = 400;  // Copies tt_content uid=1203 to first position in page uid=400
    $dataHandler->start([], $cmd);
-   $dataHandler->process-cmdmap()
+   $dataHandler->process->cmdmap()
```

Comment:
```suggestion
   $dataHandler->process_cmdmap()
```

Actually this method is called `process_cmdmap()`:

https://github.com/TYPO3/typo3/blob/6c6cb054ec2d56648fdf5d354f6da1e384678b7c/typo3/sysext/core/Classes/DataHandling/DataHandler.php#L3067

Reply:
Ups, yep you're right 🙈